js/food.js

```diff
@@ -0,0 +1,62 @@
+// XMLHttpRequests are REALLY HARD and I hope I did this in a sustainable manner
+
+// Elements for inserting information into the Dog Food Table in the DOM
+var products = document.getElementById('products');
+var dogBrands = document.getElementsByClassName('brand-dog');
+var dogType = document.getElementsByClassName('types-dog');
+
+// Nesting ALL the for-loops to get the objects to appear in the order I want in the DOM
+// STILL SURPRISED THAT THIS WORKS, WHAT KIND OF WITCHCRAFT IS THIS?!
+function putTheDogStuffintheDom () {
+  var dogObjects = JSON.parse(dogs.responseText);
+  console.log(dogObjects);
+  for (var i = 0; i < dogObjects.dog_brands.length; i++) {
+    dogBrands.item(i).innerHTML += dogObjects.dog_brands[i].name;
+    for (var j = 0; j < dogObjects.dog_brands[i].types.length; j++) {
+      dogType.item(i).innerHTML += `${dogObjects.dog_brands[i].types[j].type}: `;
+      for (var k = 0; k < dogObjects.dog_brands[i].types[j].volumes.length; k++) {
```

Comment:
...further simplify ```dogObjects.dog_brands[i].types[j].volumes.length``` to ```type.volumes.length``` etc.

As you can see, the deeper the nesting and looping, the more difference saving off the current depth item as a variable makes, cutting out a _load_ of complicated code where it's possible to stick a ```j``` instead of an ```i``` and cause it to break in weird ways.